client/blocks/inline-help/inline-help-rich-result.jsx

```diff
@@ -75,10 +75,10 @@ class InlineHelpRichResult extends Component {
 		);
 
 		this.props.recordTracksEvent( `calypso_inlinehelp_${ type }_open`, tracksData );
-		this.props.closePopover();
 
 		if ( type === RESULT_TOUR ) {
 			event.preventDefault();
+			this.props.closePopover();
```

Comment:
It looks like only tours should close the FAB, since more dialogs / modals open. But I'm still looking for a video example.

Reply:
I found a video example and it does open a similar modal to articles, so I'm leaving the FAB open when a video is clicked in these changes.